x-pack/plugins/apm/server/lib/helpers/setup_request.ts

```diff
@@ -45,23 +61,24 @@ export function setupRequest(req: Legacy.Request): Setup {
   const client: ESClient = async (type, params) => {
     const includeFrozen = await uiSettings.get('search:includeFrozen');
 
+    const nextParams = {
+      ...addFilterForLegacyData(params), // filter out pre-7.0 data
```

Comment:
Are you going to make this conditional/bypassable somehow as part of this ticket?

Reply:
No, I was not going to make the conditional in this PR.

Reply:
OK, sounds good. We might be leaning away from worrying about that for the UI messaging anyway.